### PR TITLE
Use community submitted engine to check groovy code

### PR DESCRIPTION
**Pull request by filipesperandio**

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ engines:
   pmd:
     enabled: true
     channel: beta
+  codenarc:
+    enabled: true
+    channel: beta
```
Comment (dblandin) [inline .codeclimate.yml]: That was quick! This hasn't been released yet though. Will comment here when I have an update.

Comment (dblandin) [reply .codeclimate.yml]: I'm surprised that this build didn't error, provided the unknown engine... @wfleming?

Comment (dblandin) [reply .codeclimate.yml]: Engine beta is out 👍

Comment (filipesperandio) [reply .codeclimate.yml]: I was surprised it didn't fail too. I've seen analysis failing when enabling a not released engine before. Didn't make sense to me why it didn't say anything at that time.

**Pull request merged**